conf/quincy/upi/octo-5-node-env.yaml

```diff
@@ -0,0 +1,88 @@
+# Physical/Bare-metal environment required for RBD persistent write back cache.
+# The below defined cluster has 5 nodes + 1 SSD cache node.
+# Cluster configuration:
+#      3 MONS, 2 MGR, 3 OSD nodes
+#      1 SSD Drive cache node
+globals:
+  -
+    ceph-cluster:
+      name: ceph
+      networks:
+        public: ['10.8.128.0/21']
+      nodes:
+        -
+          hostname: clara003
+          id: node1
+          ip: 10.8.129.3
+          root_password: passwd
```

Comment:
Wouldn't it cause a `security threat` as we are passing and hard coding root password here,
Can we make it self configurable from user in `cephci.yaml` template?

Reply:
Currently, all UPI tests are configured this way, until we have new provisioning.
please check this file, https://github.com/red-hat-storage/cephci/blob/master/conf/quincy/upi/octo_11_node_env.yaml